gulp-test-chrome-debug.js

```diff
@@ -1,64 +1,69 @@
 const { series } = require('gulp');
 const del = require('del');
 const run = require('gulp-run-command').default;
+const fs = require('fs');
+const extract = require('extract-zip');
+const path = require('path');
+const request = require('request');
 
-const IS_WINDOWS = (process.platform == 'win32');
+const CHROME_DEBUG_V2_URL = 'https://github.com/Microsoft/vscode-chrome-debug/archive/v2.zip';
```

Comment:
Does this just let you download any branch as a zip? Cool, but why not git clone?

Reply:
I tried for a few days to get git-clone working in azure with no luck. The docs seem to say that it's possible with the yaml, but [this issue](https://github.com/Microsoft/azure-pipelines-yaml/issues/103) seems to say otherwise. From there, I've tried within gulp to just launch git with spawn, but something seems to be messed up with the git install and/or path on the build machines, as I consistently get this error:

> git: 'remote-https' is not a git command. See 'git --help'.

Without being able to remote in to a machine somewhere and do some testing (or find a way to set up a VM or something to mirror the setup) I'm finding it really hard to debug the issue. Downloading the zip is a workaround since I don't want to spend a lot more time on this issue. Any tips on  are appreciated, though :)

Reply:
Uhg, still failing with the same issue because something inside `npm install` is trying to run a git command